src/index.js

```diff
@@ -5,17 +5,19 @@ import path from 'path'
 import 'regenerator-runtime/runtime'
 import { createElement, pdf, PDFRenderer } from '@react-pdf/core'
 import { createApolloFetch } from 'apollo-fetch'
-import Document from './components/Document'
-
-const PORT = process.env.PORT || 3007
 
 if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config()
 }
 
+const PORT = process.env.PORT || 3007
+
+const Document = require('./components/Document').default
```

Comment:
Why do we need to call `require` and not use ES6 import syntax here? Just curious  😄

Reply:
Because `require('dotenv').config()` needs to run first in development. Otherwise the env used in `RenderAsImage` might not be available yet